Created 3x3 tic-tac-toe game with android studio.

mikhailDyoshin opened the pull request.

Commit:
Created 3x3 tic-tac-toe game with android studio.

dmisuvorov commented on TicTacToeGame/.gitignore:
```diff
@@ -0,0 +1,15 @@
+/captures
+.externalNativeBuild
+.cxx
+local.properties
```
Пример моего .gitignore 
[gitignore.txt](https://github.com/mikhailDyoshin/TicTacToeAndroidGame/files/11706129/gitignore.txt)

dmisuvorov commented on TicTacToeGame/app/src/main/res/drawable/cross.png:
Советую не использовать растровую графику в ресурсах -  это оверхед на размер apk и скорость работы.
В drawable лежат обычно векторная графика. Попробуй импортировать иконки как SVG
Советую почитать: https://developer.android.com/studio/write/vector-asset-studio

dmisuvorov commented on TicTacToeGame/app/src/main/res/values/styles.xml:
Очень круто что ты выделил стили.
Для текста обычно рекомендуется использовать TextAppearence в стилях.
Как пример: https://developer.android.com/develop/ui/views/theming/themes#Styles

Commit:
Created Tic-Tac-Toe game with jetpack compose

The game has following options: different number of players (2, 3, 4), different boards (3x3, 5x5, 7x7) and two game modes (VS player and VS computer).
